docs/fast_data/connectors/debezium_connect_cdc.md

```diff
@@ -0,0 +1,162 @@
+
+Once Debezium Connect is running, you can start new conncetors by simply invoking the REST API `PUT/connectors/:connectorName/config`. This REST API will start a new connector if `connectorName` is not present, or update the configuration of an existing connector. It accepts the configuration in the request body.
+
+For a complete list of configuration parameters, refer to the [official documentation](https://debezium.io/documentation/reference/connectors/oracle.html#required-debezium-oracle-connector-configuration-properties). Note that this documentation varies for each type of source database.
+
+Here is an example configuration for an Oracle database:
+```json
+{
+  "connector.class" : "io.debezium.connector.oracle.OracleConnector",  
+  "tasks.max" : "1",  
+  "database.hostname" : "{{ORACLE_HOST}}",  
+  "database.port" : "{{ORACLE_PORT}}",  
+  "database.user" : "{{ORACLE_USER}}",  
+  "database.password" : "{{ORACLE_PASSWORD}}",   
+  "database.dbname" : "{{ORACLE_DBNAME}}",
+  "topic.prefix" : "{{INGESTION_TOPIC_PREFIX}}",
+	"topic.heartbeat.prefix": "debezium-heartbeat",
+  "snapshot.mode": "schema_only",
+  "table.include.list": "MY_SCHEMA.MY_TABLE1, MY_SCHEMA.MY_TABLE2",
+  "column.exclude.list": "MY_SCHEMA.MY_TABLE2.COL1,MY_SCHEMA.MY_TABLE2.COL2",
+  "skip.messages.without.change": true,
+  "decimal.handling.mode": "string",
+  "interval.handling.mode": "string",
+  "heartbeat.interval.ms": 5000,
+  "signal.enabled.channels": "kafka",
+	"signal.data.collection": "{{ORACLE_DBNAME}}.SVC_DEBEZIUM.DEBEZIUM_SIGNAL",
+	"signal.kafka.bootstrap.servers": "{{KAFKA_BOOTSTRAP_SERVERS}}",
+  "signal.kafka.topic": "{{INGESTION_TOPIC_PREFIX}}.signals",
+	"signal.kafka.groupId": "debezium",
+  "signal.consumer.security.protocol": "SASL_SSL",
+  "signal.consumer.sasl.mechanism": "SCRAM-SHA-256",
+  "signal.consumer.sasl.jaas.config": "{{KAFKA_SASL_JAAS_CONFIG}}",
+  "notification.enabled.channels": "sink",
+  "notification.sink.topic.name": "{{INGESTION_TOPIC_PREFIX}}.notifications",
+  "topic.transaction": "__transactions__",
+  "schema.history.internal.kafka.bootstrap.servers" : "{{KAFKA_BOOTSTRAP_SERVERS}}", 
+  "schema.history.internal.kafka.topic": "{{INGESTION_TOPIC_PREFIX}}.schema_history",
+  "schema.history.internal.producer.security.protocol": "SASL_SSL",
+  "schema.history.internal.producer.sasl.mechanism": "SCRAM-SHA-256",
+  "schema.history.internal.producer.sasl.jaas.config": "{{KAFKA_SASL_JAAS_CONFIG}}",
+  "schema.history.internal.consumer.security.protocol": "SASL_SSL",
+  "schema.history.internal.consumer.sasl.mechanism": "SCRAM-SHA-256",
+  "schema.history.internal.consumer.sasl.jaas.config": "{{KAFKA_SASL_JAAS_CONFIG}}",
+  "schema.history.internal.consumer.group.id": "debezium",
+	"schema.history.internal.store.only.captured.tables.ddl": true
+}
+```
+
+## Operate on Debezium Connect connectors
+You can easily start, stop, pause, or delete the connectors by invoking the Kafka Connect REST APIs.
```

Comment:
I would highlight the fact that this endpoint should not be exposed to the public, they should only be accessible internally with proper authorization mechanisms

Reply:
I have added a warning